src/pages/patients/Patients.tsx

```diff
@@ -123,6 +148,38 @@ const Patients = (props: PatientsProps) => {
             content={
                 <div className="flex flex-col">
                     <Search onChange={(event) => handleOnSearch(event.target.value)} size="large" placeholder={Strings.searchPatient} onSearch={handleOnSearch} enterButton />
+                    <div className="flex w-full items-end justify-end mt-4 mb-4">
+                        <Checkbox
+                        checked={filterStatus.active}
+                        onChange={(e) =>
+                            setFilterStatus({ ...filterStatus, active: e.target.checked })
+                        }
+                        >
+                        Activo
```

Comment:
can you move these activo, inactivo etc, into the Strings file? and use it as constants? maybe you can check if these values already exists

Reply:
Ok, I'll check it out

Reply:
I have already corrected it